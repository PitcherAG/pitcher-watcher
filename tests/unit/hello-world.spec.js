import helloWorld from '@/components/hello-world.vue'
import { shallowMount } from '@vue/test-utils'

describe('hello-world', () => {
  const getWrapper = () =>
    shallowMount(helloWorld, {
      propsData: {
        languages: {},
      },
    })

  describe('rendering', () => {
    it('renders correctly', () => {
      const wrapper = getWrapper()

      expect(wrapper).toMatchSnapshot()
    })
  })
})
