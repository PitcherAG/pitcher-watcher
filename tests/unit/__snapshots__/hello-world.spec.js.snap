// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`hello-world rendering renders correctly 1`] = `
<div class="hello-world-container">
  <div class="subtitle-2 mb-2">Language</div>
  <vselect-stub color="primary" errorcount="1" errormessages="" messages="" rules="" successmessages="" value="en" appendicon="$dropdown" backgroundcolor="" dense="true" hidedetails="true" loaderheight="2" clearicon="$clear" outlined="true" type="text" valuecomparator="function deepEqual(a, b) {
  if (a === b) return true;

  if (a instanceof Date &amp;&amp; b instanceof Date &amp;&amp; a.getTime() !== b.getTime()) {
    // If the values are Date, compare them as timestamps
    return false;
  }

  if (a !== Object(a) || b !== Object(b)) {
    // If the values aren't objects, they were already checked for equality
    return false;
  }

  var props = Object.keys(a);

  if (props.length !== Object.keys(b).length) {
    // Different number of props, don't bother to check
    return false;
  }

  return props.every(function (p) {
    return deepEqual(a[p], b[p]);
  });
}" nodatatext="$vuetify.noDataText" items="" itemcolor="primary" itemdisabled="disabled" itemtext="text" itemvalue="value" menuprops="[object Object]"></vselect-stub>
</div>
`;
